packages/merge-on-green/src/merge-logic.ts

```diff
@@ -669,6 +692,14 @@ export async function mergeOnGreen(
       await mergeOnGreen.merge(owner, repo, pr, prInfo, github);
       merged = true;
     } catch (err) {
+      if (err.status === 405) {
+        const isCommented = commentsOnPR?.find(element =>
+          element.body.includes('not authorized to push to this branch')
+        );
+        if (!isCommented) {
+          await mergeOnGreen.commentOnPR(owner, repo, pr, err.message, github);
```

Comment:
Looks like this still depends on the content of the API error message. Perhaps make this a `const` and use it on both lines?

Reply:
Ah! This actually depends on the content of the comments in the PR. So, really it's just checking to make sure I didn't  already put the message there before. But, you're right that that message could change, so I can make a wrapper for that message and check the wrapper so I can ensure consistency!